qutebrowser/misc/guiprocess.py

```diff
@@ -22,9 +22,11 @@
 import shlex
 
 from PyQt5.QtCore import (pyqtSlot, pyqtSignal, QObject, QProcess,
-                          QProcessEnvironment)
+                          QProcessEnvironment, QUrl)
 
-from qutebrowser.utils import message, log
+from qutebrowser.utils import message, log, objreg
+
```

Comment:
Please remove this blank line - those are only used to group Python/third-party/qutebrowser imports.